packages/@storefront/flux-capacitor/src/core/reducers/data/past-purchases.ts

```diff
@@ -31,6 +31,12 @@ export enum SORT_ENUMS {
   DEFAULT, MOST_PURCHASED, MOST_RECENT
 }
 
+export const SORT_FIELDS = [
+  'Default',
```

Comment:
Store each string as a const or something and export the whole thing at the top level (maybe put it in an object like PAST_PURCHASE_SORTS so you can export less things)

Reply:
Noted, will implement.

Reply:
Addressed, resolving.